it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -56,94 +56,94 @@
+12517=Si è verificato un errore durante l'invio della e-mail. Il server di posta ha risposto: %s. Controlla il messaggio e riprova.
 
 ## @name NS_ERROR_POST_FAILED
-12518=Il messaggio non può essere pubblicato a causa della mancata connessione al server delle news. Il server potrebbe non essere raggiungibile o rifiutare la connessione. Ricontrollare le impostazioni del server delle news o contattare il proprio amministratore di sistema.
+12518=Il messaggio non può essere pubblicato poiché la connessione al server delle news non è riuscita. Il server potrebbe non essere disponibile o rifiutare la connessione. Verifica che le tue impostazioni del server news siano corrette oppure contatta il tuo Amministratore di sistema.
 
 ## @name NS_ERROR_QUEUED_DELIVERY_FAILED
 12519=Si è verificato un errore nella consegna dei messaggi da inviare.
 
 ## @name NS_ERROR_SEND_FAILED
-12520=Non è riuscito l'invio del messaggio.
+12520=Invio del messaggio non riuscito.
 
 ## @name SMTP_DELIV_MAIL
-12521=Consegna posta in corso…
+12521=Consegna e-mail in corso…
 
 ## @name SMTP_MESSAGE_SENT_WAITING_MAIL_REPLY
-12522=Posta: Messaggio inviato; in attesa di risposta…
+12522=E-mail: Messaggio inviato; in attesa di risposta…
 
 ## @name SMTP_PROGRESS_MAILSENT
-12523=Posta inviata con successo
+12523=E-mail inviata con successo
 
 ## @name NS_ERROR_SMTP_SERVER_ERROR
-12524=Si è verificato un errore durante l'invio della posta: Errore del server SMTP. Il server ha risposto: %s Contattare l'amministratore del server di posta per assistenza.
+12524=Si è verificato un errore durante l'invio della e-mail: errore del server SMTP. Il server ha risposto: %s Contatta il tuo Amministratore e-mail per assistenza.
 
 ## @name NS_MSG_UNABLE_TO_SEND_LATER
-12525=Impossibile salvare il messaggio per l'invio ritardato.
+12525=Impossibile salvare il tuo messaggio per inviarlo più tardi.
 
 ## @name NS_ERROR_COMMUNICATIONS_ERROR
-12526=Si è verificato un errore di comunicazione: %d. Riprovare.
+12526=Si è verificato un errore di comunicazione: %d. Si prega di riprovare.
 
 ## @name NS_ERROR_BUT_DONT_SHOW_ALERT
 12527=QUESTO È SOLO UN SEGNAPOSTO. NON DOVRESTI MAI VEDERE QUESTA STRINGA.
 
 ## @name NS_ERROR_TCP_READ_ERROR
-12528=Si è verificato un errore di rete durante la ricezione dei dati. (Errore di rete: %s) Tentare di nuovo la connessione.
+12528=Si è verificato un errore di rete durante la ricezione dei dati. (Errore di rete: %s) Riprovare a connettersi di nuovo.
 
 ## @name NS_ERROR_COULD_NOT_GET_USERS_MAIL_ADDRESS
-12529=Si è verificato un errore durante l'invio della posta: l'indirizzo di posta di ritorno non era valido. Verificare che il proprio indirizzo e-mail nelle impostazioni di posta sia corretto e riprovare.
+12529=Si è verificato un errore durante l'invio della e-mail: l'indirizzo e-mail non era valido. Verifica che il tuo indirizzo e-mail sia corretto nelle tue impostazioni E-mail e riprova.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12529=Si è verificato un errore durante l'invio dell'e-mail: l'indirizzo e-mail non era valido. Verifica che il tuo indirizzo e-mail sia corretto nelle tue impostazioni e-mail e riprova.